docs/start-building.md

```diff
@@ -63,7 +66,9 @@ The following are HTTP status code your application might receive when reaching
 - HTTP 403 return code is used when the request is not authenticated.
 - HTTP 404 return code is used when the resource doesn't exist.
 - HTTP 418 return code is used when the user has been auto-banned for flooding too much after previously receiving error code 402 or 429.
-- HTTP 425 return code is used when the user has submitted a transaction when the mempool is already full, not accepting new txs straight away.
+- HTTP 425 return code
+  - in Cardano networks is used when the user has submitted a transaction when the mempool is already full, not accepting new txs straight away.
```

Comment:
this sounds weird to me, maybe adding "it" would help? "In Cardano networks, it is used..."... Also shouldn't "in" start with capital?